tools/templates/moar/Makefile.in

```diff
@@ -89,7 +89,7 @@ testclean:
 
 m-all: $(NQP_MOAR) $(M_RUNNER) $(P5QREGEX_MOAR)
 
-m-install: m-all @@configure_script@@ @@nfp(@templates_dir@/@platform@/nqp-m)@@ @@nfp(@templates_dir@/@platform@/nqp-m-reloc)@@
+m-install: m-all @@configure_script@@ @@template(@platform@/nqp-m-nonreloc)@@ @@template(@platform@/nqp-m-reloc)@@
```

Comment:
I think you could as well use @relocatable@ here instead of keeping both templates here. The final Makefile content will depend on `--relocatable` anyway. But that's just a minor thing.

Reply:
True. Done.